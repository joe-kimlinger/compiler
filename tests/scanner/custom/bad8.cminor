_this_is_an_id
this_1s_t00
thisis&andthis
#notthis
