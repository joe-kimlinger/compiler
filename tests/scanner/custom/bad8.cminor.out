IDENTIFIER _this_is_an_id
IDENTIFIER this_1s_t00
IDENTIFIER thisis
